test.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
```

Comment:
I'm not sure where this script runs. This is equivalent to `./script/go test ./go/...` (or `./script/go test -count 1 ./go/...` if you want to disable test caching)